src/resources/i18n/translations/en.json

```diff
@@ -618,10 +615,10 @@
 	"You must verify your email address to login Press Send Email then check your inbox of a verification email": "Verify your e-mail address to log in. Press \"Send e-mail\" and click it in your inbox.",
 	"You need to login to make clips": "Log in to make clips",
 	"You should receive an email shortly The email may go to your Spam folder": "You should receive an e-mail shortly. It may be in your spam.",
-	"Your free trial expired": "Your gratis Podverse premium ended ",
-	"Your free trial expires": "Your gratis Podverse period period will end ",
-	"Your membership expired": "Your Podverse premium ended ",
-	"Your premium membership expires": "Your Podverse premium will end ",
+	"Your free trial expired": "Your free trial Podverse Premium ended ",
```

Comment:
```suggestion
	"Your free trial expired": "Your Podverse Premium ended ",
```

Reply:
Ok, you've convinced me. We won't message "free trial" vs "premium membership" distinctions Can you just change it to "Your Podverse Premium membership ended"?

Reply:
Does it being a membership add anything valuable? Something is missing and the user is prompted to replace it.
It being unknown and communicated as a membership is not a good sell.

Reply:
There is no article there. You are buying Podverse Premium, proper noun. I would call it "Podverse Care".
As long as no release is made before it is figured out, it doesn't matter (as much as getting other fixes in).

Reply:
It can change later, I don't see much in it.
_Podverse_ big p _Premium_ means it is a proper noun, so "Your Podverse Premium ended" makes sense, and it is consistent between web and app right now. Otherwise it would just be "Podverse membership".
Membership sounds like it grants access to media, not premium features. That is why I don't like it.
It is important to convey something that can be understood and hopefully not misunderstood.